src/erasure-code/isa/xor_op.cc

```diff
@@ -181,3 +196,42 @@ region_sse2_xor(char** src,
 #endif // __x86_64__
   return;
 }
+
+void
+// -----------------------------------------------------------------------------
+region_neon_xor(char **src,
+                char *parity,
+                int src_size,
+                unsigned size)
+// -----------------------------------------------------------------------------
+{
+#if defined(__aarch64__) && defined(__ARM_NEON)
```

Comment:
Seems like we should ifdef out the whole definition?

Reply:
Imitate __x86_64__ behavior, could you make the question more clearly? Thanks.

Reply:
This PR leaves an empty function if not ```defined(__aarch64__) && defined(__ARM_NEON)``` -- why not remove the function entirely rather than leaving a stub?

Reply:
You are right. I just kept the same as 'region_sse2_xor', but modified now. Thanks